src/main/java/org/opensearch/sdk/sample/helloworld/spec/openapi.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
For my learning: Why do we need specs with `.json` and `.yml`? 
Is it mostly of different clients to generate APIs?

Reply:
We don't need both. It's probably an either-or.  I find the YAML one easier to read/edit, but the JSON one seems common for the OpenSearch spec.  Also there is discussion about using other tools to generate either of these from a simpler format.  This is mostly just a prompt/reminder for the future.  See #127

Reply:
(Another reason I prefer YAML: comments are supported.)

(But OpenSearch uses JSON.)

![image](https://user-images.githubusercontent.com/9291703/188935407-6f6b47ef-a1e9-4e0f-a247-d5b431aee4f5.png)